### PR TITLE
OpInfos for torch.{flatten, column_stack}

### PR DESCRIPTION
**Pull request by anjali411**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #67555
* #67355

Differential Revision: [D32650384](https://our.internmc.facebook.com/intern/diff/D32650384)

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -5759,6 +5759,28 @@ def sample_inputs_atleast1d2d3d(op_info, device, dtype, requires_grad, **kwargs)
     samples.append(SampleInput(input_list, ))
     return samples
 
+def sample_inputs_column_stack(op_info, device, dtype, requires_grad, **kwargs):
+    input_list = []
+    cases: Tuple[tuple, tuple] = (  # type: ignore[assignment]
+        ((S, 2, 1), (S, 3, 1)),
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:5765]: nit: one line per test case

File: torch/testing/_internal/common_methods_invocations.py [hunk 2]
```diff
@@ -5759,6 +5759,28 @@ def sample_inputs_atleast1d2d3d(op_info, device, dtype, requires_grad, **kwargs)
...
+        ((S), (S, 5)), ((), (1, S))
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:5766]: `((S), (S, 5))` -> `((S,), (S, 5))`?

**Review by mruberry — APPROVED** [review]
Awesome! Just a couple inline comments for your consideration.

Thanks @anjali411!

File: torch/testing/_internal/common_methods_invocations.py (hunk 1 above)
Comment (anjali411) [reply torch/testing/_internal/common_methods_invocations.py:5765]: Each test case is a list of tensors, so we already have one line per test case

Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:5765]: My mistake

**Comment by anjali411** [comment]
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32650384).

**Pull request closed without merging**